site/content/in-dev/unreleased/getting-started/minio.md

```diff
@@ -0,0 +1,114 @@
+---
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+Title: Using MinIO
+type: docs
+weight: 500
+---
+
+In this guide we walk through setting up a simple Polaris Server with local [MinIO](https://www.min.io/) storage.
+
+Similar configurations are expected to work with other S3-compatible systems that also have the
+[STS](https://docs.aws.amazon.com/STS/latest/APIReference/welcome.html) API.
+
+# Setup
+
+Start MinIO. Here's a `docker` example.
+
+```shell
+docker run -p 9100:9000 -p 9101:9001 --name minio \
+ -e "MINIO_ROOT_USER=miniouser" -e "MINIO_ROOT_PASSWORD=miniopwd" \
+ quay.io/minio/minio:latest server /data --console-address ":9001"
+```
+
+Create a bucket called "test" in MinIO UI or CLI.
+
+Edit `~/.aws/credentials` and create a `minio` profile.
+
+```
+[minio]
+aws_access_key_id = miniouser
+aws_secret_access_key = miniopwd
+region=us-west-2
+```
+
+In the shell where the Polaris Server will be started `export AWS_PROFILE=minio`.
+
+Build and run Polaris Server.
+
+```shell
+./gradlew assemble
+
+env POLARIS_BOOTSTRAP_CREDENTIALS=POLARIS,root,s3cr3t \ 
+ java -jar runtime/server/build/quarkus-app/quarkus-run.jar
+```
+
+Note: Apache Polaris 1.0.0-incubating does not yet have all the code required for proper operation with MinIO.
+The next release is expected to provide full support for S3-compatible storage with STS.
+
+Create a catalog named "polaris".
+
+```shell
+./polaris --client-id root --client-secret s3cr3t \
+ catalogs create polaris --storage-type S3 \
+ --default-base-location 's3://test' \
+ --role-arn arn:aws:iam::123456789012:role/dummy \
+ --region us-west-2 \
+ --endpoint http://127.0.0.1:9100 \
+ --path-style-access
+```
+
+Note: the role and region parameters need to be set to avoid runtime errors in Polaris,
+but they will be ignored by MinIO. 
```

Comment:
huh, that's weird. should we fix this? Should we have made a different `storage-type`?

Reply:
I previously comments under #1973 that non-AWS S3 work will address that issue, but it looks like it did not :sweat_smile: I did not find the root cause yet.

Reply:
FYI https://github.com/apache/polaris/issues/2325 will remove the need to input a role ARN.

Reply:
I'll wait for #2325 and rework this example